introduction.md

```diff
@@ -3,3 +3,11 @@
 Hi, I'm Ben Hourigan. It's nice to be a part of the dojo.
 
 My favorite CodeSchool path so far has been HTML.
+
+Deadly skills:
+
+* HTML/CSS
+* Ruby
+* Git
+* Markdown
+* English
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?